keras/backend.py

```diff
@@ -409,18 +408,16 @@ def _internal_get_learning_phase(graph):
     phase = _GRAPH_LEARNING_PHASES.get(graph, None)
     if isinstance(phase, weakref.ref):
         return phase()
-    else:
-        return phase
+    return phase
 
 
 def _default_learning_phase():
     if context.executing_eagerly():
         return 0
-    else:
-        with name_scope(""):
-            return tf.compat.v1.placeholder_with_default(
-                False, shape=(), name="keras_learning_phase"
-            )
+    with name_scope(""):
+        return tf.compat.v1.placeholder_with_default(
+            False, shape=(), name="keras_learning_phase"
+        )
 
 
 @keras_export("keras.backend.set_learning_phase")
```

Comment:
Please revert changes in this function -- it would only run in V1 graph mode, which isn't tested on GitHub CI.